snipssonos/provider/provider_player_template.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABCMeta
+
+class A_ProviderPlayerTemplate():
+    __metaclass__ = ABCMeta
+    def __init__(self):
+        raise NotImplementedError("Please implement the __init__ method")
+
+    def play_artist(self, device, name, shuffle=False):
```

Comment:
I need those to return False.
I don't want every subclass to implement it.
with the @abstract decoration It will raise an error. I don't want to raise an any error